slash/plugins/__init__.py

```diff
@@ -119,6 +121,23 @@ def is_internal_plugin(self, plugin):
         plugin_name = plugin if isinstance(plugin, str) else plugin.get_name()
         return self._installed[plugin_name].is_internal
 
+    def _is_parallel_deactivation_needed(self, plugin):
```

Comment:
I think this should be named "is_parallel_supported", which better suits the usages in this file...